src/components/AboutMe/aboutMe.css

```diff
@@ -0,0 +1,74 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap");
+
+body {
+  font-family: "Montserrat", sans-serif;
+  color: #111;
+  margin: 0;
+}
+
+.home {
+  display: flex;
+  flex-direction: row;
+  gap: 60px;
+}
+
+.main__txt {
+  display: inherit;
+  flex-direction: column;
+  margin: 60px;
+}
+
+.main__img {
+  max-width: 552px;
+  width: 100%;
+  height: 100vh;
+}
+
+.txt__title {
+  font-size: 36px;
+  font-weight: 700;
+  line-height: 44px;
+}
+
+.txt__paragraph {
+  font-size: 22px;
+  font-weight: 400;
+  line-height: 32px;
+}
+
+.link {
+  color: inherit;
+  font-weight: inherit;
+}
+
+@media (max-width: 390px) {
```

Comment:
the mobile version does not match the design
![Screenshot 2024-04-21 at 22 08 12](https://github.com/NosatskyiK/fe-react-2024/assets/115080771/79a8db18-9471-42bb-9ff4-1998b3f44dc5)